diff.py

```diff
@@ -3073,9 +3079,11 @@ def align_diffs(
         diff_lines = [
             (base, new, old if old != new else empty) for base, new, old in diff_lines
         ]
-    elif config.zeroway or config.oneway:
+    elif config.diff_type in [DiffType.SINGLE or DiffType.SINGLE_BASE]:
         meta = TableMetadata(
-            headers=(Text("CURRENT"),),
+            headers=(
+                Text("BASE" if config.diff_type == DiffType.SINGLE_BASE else "CURRENT"),
+            ),
```

Comment:
```suggestion
            headers=(Text(header),),
```
+ local variable, to avoid the ugly wrapping